src/basics/Basics.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace basics;
+
+use InvalidArgumentException;
+
+class Basics implements BasicsInterface
+{
+
+    private const FIRST = 'first';
+    private const SECOND = 'second';
+    private const THIRD = 'third';
+    private const FOURTH = 'fourth';
+
+
+    public function __construct(private BasicsValidator $validator){}
+
+    /**
+     * @throws InvalidArgumentException
+     */
+    public function getMinuteQuarter(int $minute): string
+    {
+        $this->validator->isMinutesException($minute);
+
+        return match(true){
+            $minute > 45 || $minute == 0 => self::FOURTH,
```

Comment:
Are you sure that you should return 'fourth' for 0?

Reply:
Yes, according to the tests, 0 = 60 = fourth